### PR TITLE
Reorganizes the documentation.

### PR DESCRIPTION
**Pull request by skellock**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -56,70 +31,35 @@ Reactotron on the left, demo React Native app on the right.
...
-* July 29th, 2016 - [0.8.0](https://github.com/reactotron/reactotron/releases/tag/v0.8.0)
-* July 15th, 2016 - [0.7.0](https://github.com/reactotron/reactotron/releases/tag/v0.7.0)
-* April 27th, 2016 - [0.6.1](https://github.com/reactotron/reactotron/releases/tag/v0.6.1)
-* April 24th, 2016 - [0.6.0](https://github.com/reactotron/reactotron/releases/tag/v0.6.0)
-* April 23rd, 2016 - [0.5.0](https://github.com/reactotron/reactotron/releases/tag/v0.5.0)
-* April 23rd, 2016 - [0.4.0](https://github.com/reactotron/reactotron/releases/tag/v0.4.0)
-* April 22nd, 2016 - [0.3.0](https://github.com/reactotron/reactotron/releases/tag/v0.3.0)
-* April 21st, 2016 - [0.2.0](https://github.com/reactotron/reactotron/releases/tag/v0.2.0)
-* April 20th, 2016 - [0.2.0](https://github.com/reactotron/reactotron/releases/tag/v0.1.0)
+Also, shout out to [Kevin VanGelder](https://github.com/kevinvangelder), who spawned the idea for Reactotron by saying, "Hey, you know what would be cool? A REPL. We should do that."  He singlehandedly devoured months of my spare time.
```
Comment (kevinvangelder) [inline readme.md]: ❤️

File: readme.md [hunk 2]
```diff
@@ -44,7 +19,7 @@ Use it to:
 
 You plug it into your app as a dev dependency so it adds nothing to your product builds.
 
-The app comes in two forms, which share the same [back end](https://github.com/reactotron/reactotron/tree/master/packages/reactotron-core-server):
+The app comes in two forms which share the same [guts](https://github.com/reactotron/reactotron/tree/master/packages/reactotron-core-client):
```
Comment (jamonholmgren) [inline readme.md]: Pout of approval

Comment (skellock) [inline readme.md]: Don't be sad, you made me realize I should document how things are put together!  

I created a placeholder called "What's Inside This Repo?".  It'll be my tribute to you.

File: readme.md (hunk 1 above)
Comment (jamonholmgren) [inline readme.md]: "He singlehandedly devoured months of my spare time." 😆

**Pull request merged**